crates/cxx-qt-gen/src/generator/cpp/qobject.rs

```diff
@@ -175,7 +177,7 @@ impl GeneratedCppQObject {
             let (initializer, mut blocks) = threading::generate(&qobject_idents)?;
             generated.blocks.append(&mut blocks);
             class_initializers.push(initializer);
-        // If this type has locking enabled then add generation
+            // If this type has locking enabled then add generation
```

Comment:
that looks like a fun formatting change, maybe move this comment down into the else if block below as it's supposed to be for that block

```
} else if qobject.locking {
    // If this type has locking enabled then add generation
```

Reply:
Removed